source/Calamari.Tests/ArgoCD/Git/RepositoryHelpers.cs

```diff
@@ -33,13 +33,37 @@ public static void CreateBranchIn(GitBranchName branchName, string originPath)
                                                                      Array.Empty<Commit>(),
                                                                      false);
 
-            repository.CreateBranch(MainBranchName, emptyCommit);
+            repository.CreateBranch(MainBranchName.ToFriendlyName(), emptyCommit);
 
-            repository.Refs.UpdateTarget("HEAD", cannonicalMainBranch);
+            repository.Refs.UpdateTarget("HEAD", MainBranchName.Value);
```

Comment:
:thinking: `.Value` is sort dangerous, as its not explicit in terms of "do I get refs/heads" or not.
Whereas the other functions on gitbranch (ToFriendlyName) make it explicit.